lib/ably/realtime/connection.rb

```diff
@@ -500,7 +508,7 @@ def connection_state_ttl=(val)
       # Note that this is different to the connection serial that contains the last known serial number
       # received from the server.
       #
-      # A client serial number therefore does not guarantee a message has been received, only sent.
+      # A client serial number (referred to as msgSerial in Ably) does not guarantee a message has been received, only sent.
```

Comment:
the docs refer to this as either "message serial" (eg [here](https://www.ably.io/documentation/realtime/connection#connection-state-recover-options)) or "msgSerial", rather than "client serial", maybe worth sticking with that terminology for consistency